extensions/python/CMakeLists.txt

```
@@ -81,5 +89,8 @@ else()
     )
 endif()
 
+if (NOT WIN32 AND NOT APPLE)
+    register_extension(minifi-python-lib-loader-extension "PYTHON LIB LOADER" PYTHON-LIB-LOADER-EXTENSIONS "This enables library that loads python library for python symbols")
```

I'd keep things singular. Plural naming is individual extensions is a bad pattern IMO.
```suggestion
    register_extension(minifi-python-lib-loader-extension "PYTHON LIB LOADER" PYTHON-LIB-LOADER-EXTENSION "This enables library that loads python library for python symbols")
```